2013-10-18 17:06:01 Error: [CakeLogException] Could not load class File
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Log/LogEngineCollection.php(42): LogEngineCollection::_getLogger('File')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Log/CakeLog.php(199): LogEngineCollection->load('debug', Array)
#2 /private/var/www/apps/blogapp/Config/bootstrap.php(104): CakeLog::config('debug', Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#5 /private/var/www/apps/blogapp/webroot/index.php(92): include('/private/var/ww...')
#6 {main}
2013-10-18 17:10:10 Error: [CakeLogException] Could not load class File
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Log/LogEngineCollection.php(42): LogEngineCollection::_getLogger('File')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Log/CakeLog.php(199): LogEngineCollection->load('debug', Array)
#2 /private/var/www/apps/blogapp/Config/bootstrap.php(104): CakeLog::config('debug', Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(132): require('/private/var/ww...')
#6 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(100): ShellDispatcher->_bootstrap()
#7 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(56): ShellDispatcher->_initEnvironment()
#8 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(67): ShellDispatcher->__construct(Array)
#9 /private/var/www/frameworks/cakephp/lib/Cake/Console/cake.php(51): ShellDispatcher::run(Array)
#10 {main}
2013-10-18 17:11:10 Error: [CakeLogException] Could not load class File
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Log/LogEngineCollection.php(42): LogEngineCollection::_getLogger('File')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Log/CakeLog.php(199): LogEngineCollection->load('debug', Array)
#2 /private/var/www/apps/blogapp/Config/bootstrap.php(104): CakeLog::config('debug', Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(132): require('/private/var/ww...')
#6 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(100): ShellDispatcher->_bootstrap()
#7 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(56): ShellDispatcher->_initEnvironment()
#8 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(67): ShellDispatcher->__construct(Array)
#9 /private/var/www/frameworks/cakephp/lib/Cake/Console/cake.php(51): ShellDispatcher::run(Array)
#10 {main}
2013-10-18 17:11:59 Error: [CakeLogException] Could not load class File
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Log/LogEngineCollection.php(42): LogEngineCollection::_getLogger('File')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Log/CakeLog.php(199): LogEngineCollection->load('debug', Array)
#2 /private/var/www/apps/blogapp/Config/bootstrap.php(104): CakeLog::config('debug', Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(132): require('/private/var/ww...')
#6 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(100): ShellDispatcher->_bootstrap()
#7 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(56): ShellDispatcher->_initEnvironment()
#8 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(67): ShellDispatcher->__construct(Array)
#9 /private/var/www/frameworks/cakephp/lib/Cake/Console/cake.php(51): ShellDispatcher::run(Array)
#10 {main}
2013-10-18 17:13:09 Error: [CakeLogException] Could not load class File
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Log/LogEngineCollection.php(42): LogEngineCollection::_getLogger('File')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Log/CakeLog.php(199): LogEngineCollection->load('debug', Array)
#2 /private/var/www/apps/blogapp/Config/bootstrap.php(104): CakeLog::config('debug', Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#5 /private/var/www/apps/blogapp/webroot/index.php(92): include('/private/var/ww...')
#6 {main}
2013-10-18 17:25:14 Error: [CakeLogException] Could not load class File
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Log/LogEngineCollection.php(42): LogEngineCollection::_getLogger('File')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Log/CakeLog.php(199): LogEngineCollection->load('debug', Array)
#2 /private/var/www/apps/blogapp/Config/bootstrap.php(104): CakeLog::config('debug', Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#5 /private/var/www/apps/blogapp/webroot/index.php(92): include('/private/var/ww...')
#6 {main}
2013-10-18 17:25:43 Error: [CakeLogException] Could not load class File
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Log/LogEngineCollection.php(42): LogEngineCollection::_getLogger('File')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Log/CakeLog.php(199): LogEngineCollection->load('debug', Array)
#2 /private/var/www/apps/blogapp/Config/bootstrap.php(104): CakeLog::config('debug', Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(132): require('/private/var/ww...')
#6 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(100): ShellDispatcher->_bootstrap()
#7 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(56): ShellDispatcher->_initEnvironment()
#8 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(67): ShellDispatcher->__construct(Array)
#9 /private/var/www/frameworks/cakephp/lib/Cake/Console/cake.php(51): ShellDispatcher::run(Array)
#10 {main}
2013-10-18 17:32:12 Error: [MissingPluginException] Plugin BoostCake could not be found.
Exception Attributes: array (
  'plugin' => 'BoostCake',
)
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Core/CakePlugin.php(74): CakePlugin::load('BoostCake', Array)
#1 /private/var/www/apps/blogapp/Config/bootstrap.php(77): CakePlugin::load(Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#3 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(132): require('/private/var/ww...')
#5 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(100): ShellDispatcher->_bootstrap()
#6 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(56): ShellDispatcher->_initEnvironment()
#7 /private/var/www/frameworks/cakephp/lib/Cake/Console/ShellDispatcher.php(67): ShellDispatcher->__construct(Array)
#8 /private/var/www/frameworks/cakephp/lib/Cake/Console/cake.php(51): ShellDispatcher::run(Array)
#9 {main}
2013-10-18 17:32:19 Error: [MissingPluginException] Plugin BoostCake could not be found.
Exception Attributes: array (
  'plugin' => 'BoostCake',
)
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Core/CakePlugin.php(74): CakePlugin::load('BoostCake', Array)
#1 /private/var/www/apps/blogapp/Config/bootstrap.php(77): CakePlugin::load(Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/Configure.php(94): include('/private/var/ww...')
#3 /private/var/www/frameworks/cakephp/lib/Cake/bootstrap.php(171): Configure::bootstrap(true)
#4 /private/var/www/apps/blogapp/webroot/index.php(92): include('/private/var/ww...')
#5 {main}
2013-10-21 10:53:40 Error: Fatal Error (4): syntax error, unexpected T_ECHO in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 24]
2013-10-21 10:53:40 Error: [FatalErrorException] syntax error, unexpected T_ECHO
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 24)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 10:53:44 Error: Fatal Error (4): syntax error, unexpected T_ECHO in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 24]
2013-10-21 10:53:44 Error: [FatalErrorException] syntax error, unexpected T_ECHO
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 24)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 10:54:14 Error: Fatal Error (4): syntax error, unexpected T_STRING in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 24]
2013-10-21 10:54:14 Error: [FatalErrorException] syntax error, unexpected T_STRING
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 24)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 10:54:16 Error: Fatal Error (4): syntax error, unexpected T_STRING in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 24]
2013-10-21 10:54:16 Error: [FatalErrorException] syntax error, unexpected T_STRING
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 24)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 10:54:19 Error: Fatal Error (4): syntax error, unexpected T_STRING in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 24]
2013-10-21 10:54:19 Error: [FatalErrorException] syntax error, unexpected T_STRING
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 24)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 24, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 10:56:58 Error: Fatal Error (1): Cannot access empty property in [/private/var/www/frameworks/cakephp/lib/Cake/View/View.php, line 852]
2013-10-21 10:56:58 Error: [FatalErrorException] Cannot access empty property
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Cannot access e...', '/private/var/ww...', 852)
#1 [internal function]: ErrorHandler::handleError(1, 'Cannot access e...', '/private/var/ww...', 852, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Cannot access e...', '/private/var/ww...', 852, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 13:10:35 Error: Fatal Error (4): syntax error, unexpected '<' in [/private/var/www/apps/blogapp/View/Posts/index.ctp, line 10]
2013-10-21 13:10:35 Error: [FatalErrorException] syntax error, unexpected '<'
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 10)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 10, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 10, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 15:31:04 Error: Fatal Error (4): syntax error, unexpected T_CONSTANT_ENCAPSED_STRING, expecting ')' in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 31]
2013-10-21 15:31:04 Error: [FatalErrorException] syntax error, unexpected T_CONSTANT_ENCAPSED_STRING, expecting ')'
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 31, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 31, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 16:02:14 Error: Fatal Error (4): syntax error, unexpected T_PUBLIC in [/private/var/www/apps/blogapp/Model/Post.php, line 39]
2013-10-21 16:02:14 Error: [FatalErrorException] syntax error, unexpected T_PUBLIC
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 39)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 39, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 39, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-21 16:02:51 Error: Fatal Error (4): syntax error, unexpected T_PUBLIC in [/private/var/www/apps/blogapp/Model/Post.php, line 39]
2013-10-21 16:02:51 Error: [FatalErrorException] syntax error, unexpected T_PUBLIC
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 39)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 39, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 39, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:31:22 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:31:28 Error: Fatal Error (4): syntax error, unexpected ')', expecting '&' or T_VARIABLE in [/private/var/www/apps/blogapp/Model/Post.php, line 56]
2013-10-22 13:31:28 Error: [FatalErrorException] syntax error, unexpected ')', expecting '&' or T_VARIABLE
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:32:39 Error: Fatal Error (4): syntax error, unexpected ')', expecting '&' or T_VARIABLE in [/private/var/www/apps/blogapp/Model/Post.php, line 56]
2013-10-22 13:32:39 Error: [FatalErrorException] syntax error, unexpected ')', expecting '&' or T_VARIABLE
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:33:22 Error: Fatal Error (4): syntax error, unexpected ')', expecting '&' or T_VARIABLE in [/private/var/www/apps/blogapp/Model/Post.php, line 56]
2013-10-22 13:33:22 Error: [FatalErrorException] syntax error, unexpected ')', expecting '&' or T_VARIABLE
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:34:51 Warning: Warning (2): extract() expects parameter 1 to be array, null given in [/private/var/www/frameworks/cakephp/lib/Cake/Network/CakeRequest.php, line 286]
2013-10-22 13:34:54 Error: Fatal Error (4): syntax error, unexpected ')', expecting '&' or T_VARIABLE in [/private/var/www/apps/blogapp/Model/Post.php, line 56]
2013-10-22 13:34:54 Error: [FatalErrorException] syntax error, unexpected ')', expecting '&' or T_VARIABLE
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:36:33 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 125]
2013-10-22 13:36:33 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 125)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 125, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 125, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:39:08 Error: Fatal Error (4): syntax error, unexpected T_PUBLIC in [/private/var/www/apps/blogapp/Model/Post.php, line 69]
2013-10-22 13:39:08 Error: [FatalErrorException] syntax error, unexpected T_PUBLIC
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 69)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 69, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 69, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:39:16 Error: Fatal Error (4): syntax error, unexpected T_PUBLIC in [/private/var/www/apps/blogapp/Model/Post.php, line 69]
2013-10-22 13:39:16 Error: [FatalErrorException] syntax error, unexpected T_PUBLIC
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 69)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 69, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 69, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:39:18 Error: Fatal Error (4): syntax error, unexpected T_PUBLIC in [/private/var/www/apps/blogapp/Model/Post.php, line 69]
2013-10-22 13:39:18 Error: [FatalErrorException] syntax error, unexpected T_PUBLIC
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 69)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 69, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 69, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:41:02 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:41:02 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:43:15 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:43:15 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:43:16 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:43:16 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:44:15 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:44:15 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:44:17 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:44:17 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:45:28 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:45:28 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:48:34 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:48:34 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:48:37 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:48:37 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:48:38 Error: Fatal Error (1): Call to undefined function setFlash() in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 127]
2013-10-22 13:48:38 Error: [FatalErrorException] Call to undefined function setFlash()
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/private/var/ww...', 127)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/private/var/ww...', 127, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 13:53:13 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:54:02 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:54:03 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:54:04 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:54:04 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:56:08 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:56:10 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:56:10 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:58:12 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:58:14 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:59:44 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 13:59:45 Error: [MissingControllerException] Controller class AddController could not be found.
Exception Attributes: array (
  'class' => 'AddController',
  'plugin' => NULL,
)
Request URL: /blogapp/add
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 15:44:33 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-22 15:44:33 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 15:44:46 Error: [MissingControllerException] Controller class TagController could not be found.
Exception Attributes: array (
  'class' => 'TagController',
  'plugin' => NULL,
)
Request URL: /blogapp/tag
Stack Trace:
#0 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-10-22 15:44:49 Error: [MissingActionException] Action TagsController::index() could not be found.
Exception Attributes: array (
  'controller' => 'TagsController',
  'action' => 'index',
)
Request URL: /blogapp/tags
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#1 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(TagsController), Object(CakeRequest), Object(CakeResponse))
#2 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-10-22 15:54:45 Error: [MissingHelperException] Helper class postsHelper could not be found.
Exception Attributes: array (
  'class' => 'postsHelper',
  'plugin' => false,
)
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/View/HelperCollection.php(63): HelperCollection->load('posts')
#1 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(833): HelperCollection->__isset('posts')
#2 /private/var/www/apps/blogapp/View/Posts/edit.ctp(4): View->__get('posts')
#3 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(945): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(907): View->_evaluate('/private/var/ww...', Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(471): View->_render('/private/var/ww...')
#6 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(952): View->render(NULL, NULL)
#7 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#8 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#9 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2013-10-22 15:55:01 Error: [MissingHelperException] Helper class postHelper could not be found.
Exception Attributes: array (
  'class' => 'postHelper',
  'plugin' => false,
)
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/View/HelperCollection.php(63): HelperCollection->load('post')
#1 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(833): HelperCollection->__isset('post')
#2 /private/var/www/apps/blogapp/View/Posts/edit.ctp(4): View->__get('post')
#3 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(945): include('/private/var/ww...')
#4 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(907): View->_evaluate('/private/var/ww...', Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/View/View.php(471): View->_render('/private/var/ww...')
#6 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(952): View->render(NULL, NULL)
#7 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#8 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#9 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2013-10-22 15:56:13 Error: Fatal Error (1): Function name must be a string in [/private/var/www/apps/blogapp/Controller/PostsController.php, line 144]
2013-10-22 15:56:13 Error: [FatalErrorException] Function name must be a string
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Function name m...', '/private/var/ww...', 144)
#1 [internal function]: ErrorHandler::handleError(1, 'Function name m...', '/private/var/ww...', 144, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Function name m...', '/private/var/ww...', 144, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-22 15:57:52 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-22 15:58:04 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-22 15:58:06 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-22 15:58:58 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-22 15:59:18 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-22 15:59:19 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-22 15:59:20 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-22 15:59:33 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Post.' in 'field list'
Request URL: /blogapp/posts/edit/52669db8-ae60-49d9-920b-44a3b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT DISTINCT...', Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(1077): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(2700): DboSource->read(Object(Post), Array)
#5 /private/var/www/frameworks/cakephp/lib/Cake/Model/Datasource/DboSource.php(599): Model->find('first', Array)
#6 /private/var/www/frameworks/cakephp/lib/Cake/Model/Model.php(799): DboSource->query('findById', Array, Object(Post))
#7 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Model->__call('findById', Array)
#8 /private/var/www/apps/blogapp/Controller/PostsController.php(147): Post->findById('52669db8-ae60-4...', Array)
#9 [internal function]: PostsController->edit('52669db8-ae60-4...')
#10 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#11 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#12 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#13 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2013-10-30 13:54:45 Error: Fatal Error (4): syntax error, unexpected '(', expecting ')' in [/private/var/www/apps/blogapp/Model/Post.php, line 41]
2013-10-30 13:54:45 Error: [FatalErrorException] syntax error, unexpected '(', expecting ')'
Request URL: /blogapp/posts
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 41)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 41, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 41, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:33:13 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:33:13 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:33:16 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:33:16 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:35:29 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:35:29 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:35:30 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:35:30 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:35:30 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:35:30 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:39:49 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:39:49 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:39:50 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:39:50 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:43:08 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:43:08 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:43:09 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:43:09 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:43:09 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:43:09 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:45:46 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:45:46 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:45:47 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:45:47 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:45:48 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:45:48 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:49:33 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-10-30 16:49:33 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:51:09 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 55]
2013-10-30 16:51:09 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 55)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 55, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 55, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 16:51:13 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 55]
2013-10-30 16:51:13 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 55)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 55, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 55, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 17:05:27 Error: Fatal Error (4): syntax error, unexpected '(', expecting ')' in [/private/var/www/apps/blogapp/Model/Post.php, line 45]
2013-10-30 17:05:27 Error: [FatalErrorException] syntax error, unexpected '(', expecting ')'
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 45)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 17:37:13 Error: Fatal Error (4): syntax error, unexpected '(', expecting ')' in [/private/var/www/apps/blogapp/Model/Post.php, line 45]
2013-10-30 17:37:13 Error: [FatalErrorException] syntax error, unexpected '(', expecting ')'
Request URL: /blogapp/posts/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 45)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 17:49:28 Error: Fatal Error (4): syntax error, unexpected '(', expecting ')' in [/private/var/www/apps/blogapp/Model/Post.php, line 45]
2013-10-30 17:49:28 Error: [FatalErrorException] syntax error, unexpected '(', expecting ')'
Request URL: /blogapp/posts/view/52712cab-cd38-47d2-91ba-f1d8b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 45)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-10-30 17:50:01 Error: Fatal Error (4): syntax error, unexpected '(', expecting ')' in [/private/var/www/apps/blogapp/Model/Post.php, line 45]
2013-10-30 17:50:01 Error: [FatalErrorException] syntax error, unexpected '(', expecting ')'
Request URL: /blogapp/posts/view/52712cab-cd38-47d2-91ba-f1d8b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 45)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 13:56:02 Error: Fatal Error (4): syntax error, unexpected '(', expecting ')' in [/private/var/www/apps/blogapp/Model/Post.php, line 45]
2013-11-06 13:56:02 Error: [FatalErrorException] syntax error, unexpected '(', expecting ')'
Request URL: /blogapp/
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 45)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 45, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 13:57:40 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-11-06 13:57:40 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:01:27 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:01:27 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:01:48 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:01:48 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:05:40 Error: [NotFoundException] Id Not Found
Request URL: /blogapp/posts/view/527a4ac0-8ba0-4202-b57d-0785b47db738
Stack Trace:
#0 [internal function]: PostsController->view('527a4ac0-8ba0-4...')
#1 /private/var/www/frameworks/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#3 /private/var/www/frameworks/cakephp/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest), Object(CakeResponse))
#4 /private/var/www/apps/blogapp/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-11-06 14:21:02 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:21:02 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:23:52 Error: Fatal Error (4): syntax error, unexpected T_ECHO in [/private/var/www/apps/blogapp/View/Posts/view.ctp, line 6]
2013-11-06 14:23:52 Error: [FatalErrorException] syntax error, unexpected T_ECHO
Request URL: /blogapp/posts/view/527a4d4a-d668-475f-a730-27f1b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 6)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 6, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 6, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:24:08 Error: Fatal Error (4): syntax error, unexpected T_ECHO in [/private/var/www/apps/blogapp/View/Posts/view.ctp, line 6]
2013-11-06 14:24:08 Error: [FatalErrorException] syntax error, unexpected T_ECHO
Request URL: /blogapp/posts/view/527a4d4a-d668-475f-a730-27f1b47db738
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 6)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 6, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 6, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:40:18 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:40:18 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:43:24 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-11-06 14:43:24 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:44:40 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 56]
2013-11-06 14:44:40 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 56)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 56, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:46:28 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:46:28 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:46:35 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:46:35 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:46:36 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:46:36 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-11-06 14:46:36 Error: Fatal Error (4): syntax error, unexpected ';' in [/private/var/www/apps/blogapp/Controller/CommentsController.php, line 57]
2013-11-06 14:46:36 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /blogapp/comments/add
Stack Trace:
#0 /private/var/www/frameworks/cakephp/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/private/var/ww...', 57)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#2 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/private/var/ww...', 57, Array)
#3 /private/var/www/frameworks/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
